app/assets/javascripts/passengers/passengers.js

```diff
@@ -7,6 +7,17 @@ $( document ).on("turbolinks:load", function() {
     $('.doctors-note-fields').toggle(!$(this).prop('checked'));
   });
 
+  new ClipboardJS('#copybtn', {
+    text: function() {
+      var arr = $("tbody tr")
```

Comment:
Perhaps a selector that specifies the _passengers_ table?

Reply:
I think you can do this in a few fewer steps:

```js
text: function() {
  $("tbody tr").map(function(){return $(this).data("email")}).get().join(";")
}
```

Reply:
This doesn't seem to work though.

Reply:
Sorry. It's missing the `return`:

```js
    text: function() {
      return $("#passengers tbody tr").map(function(){
        return $(this).data("email")
      }).get().join(";")
```